src/utils/legacyLayerParamUtils.js

```diff
@@ -171,3 +171,22 @@ export async function getKmlLayerFromLegacyAdminIdParam(adminId) {
         kmlMetaData
     )
 }
+
+export function handleBodLayerIdParam(params, store, newQuery) {
+    params.forEach((param_value, param_key) => {
+        let layer = null
+        layer = store.state.layers.config.find((layer) => layer.getID() === param_key)
```

Comment:
inline const would work too
`const layer = store.state.layers.config.find((layer) => layer.getID() === param_key)`
and be careful here when rebasing, `getID()` has been replaced by `id` param (no function call anymore)

Reply:
As you don't use this `layer` constant after the `if` block, you could even replace this by
`if (store.state.layers.config.some((layer) => layer.id === param_key)) {`
and get away without a constant

Reply:
a little comment/reminder on why we get the info this way in a legacy URL might help too (in 6 month, we will have dumped the help, and won't have any trace left of this way of describing a feature pre-selection in our official doc)

Reply:
those are three valid points